init.c

```diff
@@ -1253,6 +1253,12 @@ static enum CommandResult parse_ifdef(struct Buffer *buf, struct Buffer *s,
     res = !!myvar_get(buf->data);
   }
 
+  /* or an environment variable? */
+  if (!res)
+  {
+    res = !!mutt_str_getenv(buf->data);
```

Comment:
We can combine this `if` statement with the previous one so that it is:

```C
/* or a my_ var or environment variable? */
  if (!res)
  {
    res = !!myvar_get(buf->data) || !!mutt_str_getenv(buf->data);
  }
```

Saves some duplication

Reply:
done